src/features/flows/nRF9151/SimCard.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+
+import { Back } from '../../../common/Back';
+import Link from '../../../common/Link';
+import Main from '../../../common/Main';
+import { Next } from '../../../common/Next';
+
+const SIMCard = () => (
+    <Main>
+        <Main.Content heading="Plug in SIM card">
+            <p>There are two pre-activated SIM cards in the kit:</p>
+            <br />
+            <ol className="tw-list-outside tw-list-disc tw-pl-4">
+                <li>
+                    <Link
+                        label="SIM card from Onomondo"
+                        href="https://onomondo.com/go/nordic-dev-kit/#form"
+                        color="tw-text-primary"
+                    />{' '}
+                    with 10 MB free data.
```

Comment:
activated with 10 MB of free data.

Reply:
It does talk about the sim cards being "pre-activated" above the 2 sim card entries.
Not sure if we should duplicate this mention?

Reply:
true, avoid this